src/assets/scripts/client/base/DynamicPositionModel.js

```diff
@@ -216,7 +199,7 @@ export default class DynamicPositionModel {
         const x = Math.cos(φ1) * Math.sin(φ2) - Math.sin(φ1) * Math.cos(φ2) * Math.cos(Δλ);
         const θ = Math.atan2(y, x);
 
-        return radians_normalize(θ - this._magneticNorth);
+        return radians_normalize(θ - Compass.magneticNorth);
```

Comment:
this should/could live within the `Compass`

Reply:
done